### ISSUE TITLE
Default hxcpp_config.xml with HXCPP_COMPILE_CACHE set?

### ISSUE THREAD

**Issue by thomasuster** [HaxeFoundation/hxcpp#731]
I think it may be common for people to not know about the HXCPP_COMPILE_CACHE and get frustrated with build times when switching projects. Can we have either a default hxcpp_config, or a default location for HXCPP_COMPILE_CACHE?

**Comment by hughsando** [comment]
Yes, it is well out of experimental phase now, and should be enabled by default.

**Comment by thomasuster** [comment]
Do we just need to add it to `hxcpp/toolchain/common-defines.xml`?

**Comment by hughsando** [comment]
It should go in example.hxcpp_config.xml so new people get it.
We could print a warning from the tool if it is not set, and allow setting it to empty ("") to disable the warning while also disabling the cache.
